test/logger-helper-test.js

```diff
@@ -1038,15 +1038,15 @@ describe('logger-helpers tests', function () {
                     body: 'body',
                     test1: 'test2'
                 };
-                response.headers['content-type'] = undefined;
-                response._headers['content-type'] = undefined;
+                delete response.headers['content-type'] ;
+                delete response._headers['content-type'] ;
```

Comment:
Are we sure this will not also generate a warning?
Wouldn't it be better to use
```js
response.removeHeader('content-type');
```
if the method `removeHeader` exists?

Reply:
I guess the mock module that represents the HTTP object does not really interact with the actual HTTP module. After your comment, I tried to access the same property (_header) for reading purposes also but it hasn't complained. But like you said in some other comments, we shouldn't involve with the internal method or properties anyway.  I'm not sure if `res.headers` was something that existed in the past and was deprecated later also.  So, I'm going to update similar usages in the whole test file. Thank you.